src/data/slice.js

```diff
@@ -9,6 +9,7 @@ export const examSlice = createSlice({
     activeAttempt: null,
     proctoringSettings: {},
     exam: {},
+    verification: {},
```

Comment:
We will create tests for redux later and I believe we will have snapshots / factories for our data, will it work or do you mean something else?

Reply:
I'd hoped to see something more like this for each model
```
verification: {
      status: '',
      can_verify: false,
    },
...
```

let's follow up in another PR since I've merged this already.